src/renderer/pages/player-page.js

```diff
@@ -56,6 +54,14 @@ function renderMedia (state) {
       mediaElement.pause()
     } else if (!state.playing.isPaused && mediaElement.paused) {
       mediaElement.play()
+        .then(() => {
+          dispatch('mediaSuccess')
+        })
+        .catch((err) => {
+          if (err.name === 'NotSupportedError') {
+            dispatch('mediaError', 'Codec unsupported')
+          }
```

Comment:
Maybe remove the condition and display any error?

Reply:
Yes! In our case all the road lead to `Codec unsupported` 😅